pages/dr9/sga.rst

```diff
@@ -10,178 +10,866 @@
+   ),
+   "IR" AS (
+     SELECT pgc, avg(lax) AS lax, avg(sax) AS sax
+     FROM rawdia
+     WHERE quality=0 and iref in (27129) and dcode=7 and band=0 GROUP BY pgc
+   )
+   
+   SELECT
+   count(*)
+     m.pgc, m.objname, m.objtype, m.al2000, m.de2000, m.type, m.bar, m.ring, m.multiple, m.compactness, m.t, 
+     m.logd25, m.logr25, m.pa, m.bt, m.it, m.kt, m.v, m.modbest, "R50".lax, "R50".sax, "IR".lax, "IR".sax,
+   FROM
+     m000 AS m
+     LEFT JOIN "R50" USING (pgc)
+     LEFT JOIN "IR" USING (pgc)
+   WHERE
+     objtype='G'
+     and (
+       m.logd25>0.2 or "R50".lax>0.2 or "IR".lax>0.2
+     )
+
+
+Based on a large number of visual inspections and both quantitative and
+qualitative tests, we culled the resulting sample by applying the following
+additional cuts:
+
+* First, we limited the sample to :math:`0.333<D(25)<180\ \mathrm{arcmin}`,
+  which removed roughly 900,000 galaxies (:math:`\approx63\%` of the original
+  sample), including the Magellanic Clouds and the Sagittarius Dwarf Galaxy at
+  the large-diameter end). We implemented the :math:`D(25)<20\ \mathrm{arcsec}`
+  cut because we found that the fraction of spurious sources (or sources with
+  incorrect diameters) increased rapidly below this diameter; moreover, galaxies
+  smaller than this size are modeled reasonably well as part of the standard
+  ``Tractor`` pipeline used in `DR9`_.
+  
+..  
+  
+
+* Next, we removed :math:`\approx3800` galaxies with no magnitude estimate in
+  `Hyperleda`_, galaxies which we found to be largely spurious based on visual
+  inspection.
+  
+..  
+  
+
+* Third, we removed :math:`\approx6500` spurious sources (or galaxies with
+  significantly overestimated diameters) based on visual inspection.
+  
+..  
+  
+
+* Finally, we removed :math:`\approx1700` galaxies whose primary galaxy
+  identifier (in `Hyperleda`_) was from either SDSS or 2MASS and whose central
+  coordinates placed it *inside* the elliptical aperture of *another* (non-SDSS
+  and non-2MASS) galaxy with diameter greater than 0.5 arcmin. Based on visual
+  inspection, we found that many of these sources were due to shredding or were
+  spurious sources with grossly over-estimated diameters.
+
+In addition, we visually inspected all galaxies in the sample with
+:math:`D(25)>0.75\ \mathrm{arcmin}`, including all the NGC/IC galaxies, and
+assessed their published elliptical geometry and coordinates. Where necessary,
+we updated the diameter, position angle, minor-to-major axis ratio, and, in some
+cases, central coordinates "by hand", as indicated in the `BYHAND` column
+described in the data model below. The `NASA Extragalactic Database (NED)`_
+proved invaluable for these cross-checks.
+
+``Supplemental Catalogs``
+-------------------------
+
+To improve the completeness of the `Hyperleda`_ catalog, we supplemented the
+sample with several additional catalogs:
+
+* First, we added the sample of Local Group Dwarf Galaxies from `McConnachie
```

Comment:
Similar to the comment above, why not a numbered list that is not so widely spaced out with extraneous comments?

Reply:
I preferred the look of a bulleted list but since this is an issue I've changed this to a numbered list. 

The extra space between the items of the list were because I wanted extra space between each item---the default formatting is far too "scrunched.'

But I don't understand the comment about "extraneous comments". Which portion of the documentation I've written is extraneous and should be removed?